cli/medperf/config.py

```diff
@@ -70,3 +70,10 @@
     "platform",
     "cleanup",
 ]
+
+github_repository = "https://github.com/aristizabal95/medperf-2"
+templates = {
```

Comment:
Is there a reason for using a repo? (which needs internet connection)
we can specify the folder: `abspath(Path(__file__).parent.parent.parent)`

Reply:
There is actually! We expect people in the future will install medperf with `pip install medperf` instead of cloning the repository. In those cases, the templates would not be copied, so providing the repository covers those cases.

Reply:
Maybe we can add the templates to the package folders, and make the change you suggested. I'll see if that's possible

Reply:
That absolutely worked, great idea Hasan!